client/src/components/Workflow/Editor/Forms/FormDefault.vue

```diff
@@ -27,15 +27,13 @@
         <template v-slot:body>
             <FormElement
                 id="__label"
-                :key="stepId"
```

Comment:
I seem to recollect that you need the key to avoid having old values stick around after you switch the active node.
`:key="`${step_id}_label"` and `:key="`${step_id}_annotation"` should also avoid the key error, I think.

Reply:
https://github.com/galaxyproject/galaxy/commit/a3962349f74a7337a29b64bc2e328d2d40e54605 that's 2 weeks ago 😆. I think I added this because I saw the text sticking around when you were switching between non-tool steps as the active element.

Reply:
Managed to reproduce it for sub-workflows. The issue was label and annotation not being reactive, when the step they were constructed from did not define them. Making them reactive even when undefined by using computed fixed the issue.

Reply:
excellent, thank you!